main.tf

```
@@ -38,8 +38,9 @@ resource "aws_db_parameter_group" "this" {
     for_each = var.parameters
 
     content {
-      name  = each.value.name
-      value = each.value.value
+      apply_method = each.value.apply_method
+      name         = each.value.name
+      value        = each.value.value
```

While we do alphabetize, Name is supposed to come first

```suggestion
      name         = each.value.name
      apply_method = each.value.apply_method
      value        = each.value.value
```